src/containers/card/actions.js

```diff
@@ -0,0 +1,168 @@
+import { CLEAN_STATE,
+  GET_ALL_COMMENTS_IN_CARD, GET_ALL_LABELS_IN_CARD, GET_ALL_ASSIGNEES_IN_CARD, GET_RESPONSIBLE_FOR_CARD, GET_ONE_CARD,
+  ADD_COMMENT, ADD_LABEL, ADD_MEMBER, ADD_RESPONSIBLE,
+  UPDATE_DUE_DATE, UPDATE_DESC } from './constants'
+
+export const cleanState = () => (dispatch) => {
+  dispatch({
+    type: CLEAN_STATE,
+  })
+}
+
+export const getAllCommentsInBoard = cardId => (dispatch) => {
+  dispatch({
+    type: GET_ALL_COMMENTS_IN_CARD,
+    payload: {
+      request: {
+        method: 'GET',
+        url: `/api/cards/${cardId}/comments`,
+      },
+    },
+  })
+}
+
+export const getAllAssigneesInBoard = cardId => (dispatch) => {
+  dispatch({
+    type: GET_ALL_ASSIGNEES_IN_CARD,
+    payload: {
+      request: {
+        method: 'GET',
+        url: `/api/cards/${cardId}/members`,
```

Comment:
`members` are assignees + responsible ? Why do you don't hit the specific route `/api/cards/${cardId}/assignees` ?